KerbalVR_Mod/KerbalVR-MAS/MASLever.cs

```diff
@@ -51,9 +55,36 @@ public MASLever(MASComponent component, VRLever vrLever)
 
 			flightComputer = vrLever.part.GetComponent<MASFlightComputer>();
 
+			if (lever.handler.StartsWith("CustomAxis"))
+			{
+				if (int.TryParse(lever.handler.Remove(0, 10), out int result))
+				{
+					customAxisNumber = result - 1;
+					FlightInputHandler.OnRawAxisInput += OnRawAxisInput;
+				}
+				else
+				{
+					Utils.LogError($"Invalid custom axis name '{lever.handler}' on {lever.internalProp.propName}");
+				}
+			}
+
 			lastStep = RefreshState();
 		}
 
+		~MASLever()
```

Comment:
pretty sure this will always leak memory: https://www.geeksforgeeks.org/destructors-in-c-sharp/amp/

If FlightInputHandler has a reference to this object then it will never be garbage collected.

Need to either have a virtual OnDestroy called from the owning VRLever or move the custom axis event to VRLever and have a virtual function in IVALever to fill out the control state.